### ISSUE TITLE
Make tracing headers configurable

### ISSUE THREAD

**Issue by frankgreco** [jaegertracing/jaeger-client-node#139]
Currently, `TextMapCodec ` allows dynamic configuration of `contextKey` and `baggagePrefix`. However, there seems to be no way for a user to configure these values as `TextMapCodec` is [instantiated](https://github.com/uber/jaeger-client-node/blob/6d7544087e148072fcb8bf6754772a4781d90929/src/tracer.js#L76-L79) as follows:

```js
let textCodec = new TextMapCodec({
    urlEncoding: false,
    metrics: this._metrics
});
```

This is also an open issue with the go client [here](https://github.com/uber/jaeger-client-go/issues/109).

**Comment by astub** [comment]
I'm going to open a PR for this soon.

**Comment by yurishkuro** [comment]
Fixed by #217
